opensrp-reveal/src/main/java/org/smartregister/reveal/view/ListTasksActivity.java

```diff
@@ -308,16 +319,29 @@ public void setGeoJsonSource(@NonNull FeatureCollection featureCollection, Featu
             geoJsonSource.setGeoJson(featureCollection);
             if (operationalArea != null) {
                 CameraPosition cameraPosition = mMapboxMap.getCameraForGeometry(operationalArea.geometry());
+                if (getInterventionLabel() == R.string.focus_investigation) {
+                    Feature indexCase = getIndexCase(mMapboxMap.getStyle().getSourceAs(getContext().getString(R.string.reveal_datasource_name)));
+                    if (indexCase != null) {
+                        cameraPosition = mMapboxMap.getCameraForGeometry(indexCase.geometry());
```

Comment:
Use  CameraPosition Builder to construct a camera position with target `indexCase` cordinates and specify a zoom. Level. This method would zoom to very low zoom level where the map is not clearly visible and would force user to zoom out

Reply:
Done.